src/main/java/org/radarcns/managementportal/Source.java

```diff
@@ -49,17 +49,17 @@ protected Source(){}
 
     protected Source(
```

Comment:
Since the fields are not marked `final`, i think this constructor is not necessary since the fields already have the `@JsonProperty` annotation.